src/test/java/uk/gov/hmcts/sptribs/common/service/AuthorisationServiceTest.java

```diff
@@ -0,0 +1,78 @@
+package uk.gov.hmcts.sptribs.common.service;
+
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+import uk.gov.hmcts.reform.authorisation.generators.AuthTokenGenerator;
+import uk.gov.hmcts.reform.idam.client.models.User;
+import uk.gov.hmcts.sptribs.idam.IdamService;
+
+import javax.servlet.http.HttpServletRequest;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.Mockito.when;
+import static org.springframework.http.HttpHeaders.AUTHORIZATION;
+import static uk.gov.hmcts.sptribs.testutil.TestConstants.TEST_AUTHORIZATION_TOKEN;
+import static uk.gov.hmcts.sptribs.testutil.TestConstants.TEST_SERVICE_AUTH_TOKEN;
+
+@ExtendWith(MockitoExtension.class)
+public class AuthorisationServiceTest {
```

Comment:
Not required if supplementary data change is not related to case linking. Revert back.

Reply:
Reverted this file.